docs/victory-pie/docs.js

```diff
@@ -1,25 +1,24 @@
 import React from "react";
 import ReactDOM from "react-dom";
 import Ecology from "ecology";
-import Radium, { Style, StyleRoot } from "radium";
+import Radium from "radium";
 import * as docgen from "react-docgen";
 import { merge, random, range } from "lodash";
-import { VictoryPie } from "../src/index";
-import { VictoryTheme, appendLinkIcon, ecologyPlaygroundLoading } from "formidable-landers";
+import { VictoryPie } from "../../src/index";
+import { appendLinkIcon, ecologyPlaygroundLoading } from "formidable-landers";
 
 class Docs extends React.Component {
   render() {
     return (
-      <StyleRoot>
+      <div>
```

Comment:
I don't believe we need this wrapper div anymore now that we're only rendering one component. Styles should be effected by this change, as we're using ecology's `.interactive` as the base for styling the playgrounds